src/panels/lovelace/create-element/create-card-element.ts

```diff
@@ -51,14 +51,18 @@ const LAZY_LOAD_TYPES = {
   picture: () => import("../cards/hui-picture-card"),
 };
 
-export const createCardElement = (config: LovelaceCardConfig) =>
+export const createCardElement = (
+  config: LovelaceCardConfig,
+  throwException: boolean = false
```

Comment:
We should throw the exception by default if we go this route and only not throw exceptions when specifically stated.

Reply:
Currently, when there is an exception when create the card element, the error will be printed to the console and it will create and return an error element using `_createErrorElement` so that it can be displayed on UI. If we throw an exception by default, the error element will not be created and there is no way to know the error message on the UI.